modules/benchmark/R/get_species_list_standard.R

```diff
@@ -0,0 +1,32 @@
+#'@title get_species_list_standard
+#' Checks if custom_table:
+#' 1. is formated correctly
+#' 2. is complete (has all of the species/pft's in both observations)
+#' 3. is condense-able (Could be represented as a hierachry)
+#' 
+#' @param observation_one a vector of plant fucntional types, or species
+#' @param observation_two anouther vector of plant fucntional types, or species
+#' @param custom_table a table that either maps two pft's to one anouther or maps custom species codes to bety id codes. 
+#' In the second case, must be passable to match_species_id. 
+#' @return \code{character} Returns "usda", "latin_name", "fia" or "custom"
+#' @author Tempest McCabe
+get_species_list_standard<-function(vars){
+  
+  if("species_id" %in% vars){
+    return("usda")
+  }else if("species_name" %in% vars){
+    return('latin_name')
+  }else if("species_USDA_symbol" %in% vars){
+    return("usda")
+  }else if("species_FIA_symbol" %in% vars){
+    return('fia')
```

Comment:
Ditto here. Besides being more efficient under the hood (which effectively doesn't matter at all here), a code structure like:

```r
if (any(c("a", "b", "c", "d", "e") %in% vars)) ...
```

...is much easier to handle if, for example, sometime in the future you decide to change `vars` to some other name. This way, you only have to change it once, rather than having to change it multiple times, which is much more prone to forgetting it somewhere.